docs/platform-services/automation-service/app-central/integrations/sumo-logic-automation-tools.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Sumo Logic Automation Tools
+description: ''
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('/img/platform-services/automation-service/app-central/logos/sumo-logic-notifications.png')} alt="sumo-logic-notifications" width="100"/>
+
+***Version: 1.0  
+Updated: Sep 9, 2024***
+
+Sumo Logic Automation Tools simplifies CloudSOAR playbooks with data processing and automation.
+
+### Actions
+
+* **Buffer** (*Custom*) - [Description](#buffer)
+* **Data Transform** (*Custom*) - [Description](#data-transform)
+* **Build JSON Object** (*Custom*) - [Description](#build-json-object)
+* **Build Signal Output** (*Custom*) - [Description](#build-signal-output)
+
+## Actions Details
+
+### Buffer
```

Comment:
Buffer does not seem to be the right name for the functionality this action is doing. Can you figure out some alternate names ?